modules/tinymce/src/core/main/ts/init/Render.ts

```diff
@@ -99,6 +100,8 @@ const loadIcons = (scriptLoader: ScriptLoader, editor: Editor, suffix: string) =
 };
 
 const loadPlugins = (editor: Editor, suffix: string) => {
+  const platform = PlatformDetection.detect();
```

Comment:
This could use the `Env` module instead as it will produce smaller code once bundled/minified.

Reply:
To clarify, I mean this should just use `Env.browser.isIE()`.

Reply:
Yeah, that would be nice.